website/docs/reference/runtimes/index.mdx

```diff
@@ -15,14 +17,41 @@ The status represents the maturity of the runtime implementation:
 Submit your request and vote for your preferred ones [here](https://github.com/metatypedev/metatype/discussions/305).
 :::
 
-| Runtime                         | Description                                                | Runtime version | Status | Typegate version |
+|             Runtime             |                        Description                         | Runtime version | Status | Typegate version |
 | ------------------------------- | ---------------------------------------------------------- | --------------- | ------ | ---------------- |
 | [Prisma](./runtimes/prisma)     | Query SQL and no-SQL databases with Prisma                 | 4.17.0          | beta   | >0.1.0           |
 | [Deno](./runtimes/deno)         | Execute Typescript functions with NPM or deno dependencies | 1.35.0          | beta   | >0.1.0           |
 | [HTTP](./runtimes/http)         | Make HTTP requests to (REST) HTTP APIs                     |                 | beta   | >0.1.0           |
 | [GraphQL](./runtimes/graphql)   | Make GraphQL requests                                      |                 | beta   | >0.1.0           |
 | [Random](./runtimes/random)     | Generate random data based on your schema                  |                 | alpha  | >0.1.0           |
 | [S3](./runtimes/s3)             | Manipulate S3-compatible storage                           |                 | alpha  | >0.1.0           |
-| [Temporal](./runtimes/temporal) | Query and trigger workflow orchestrated by Temporal        |                 | alpha  | >0.1.0           |
-| [WasmEdge](./runtimes/wasmedge) | Execute WebAssembly functions                              |                 | alpha  | >0.1.0           |
-| [Python](./runtimes/python)     | Execute Python functions                                   |                 | alpha  | >0.1.0           |
+| [Temporal](./runtimes/temporal) | Query and trigger workflow orchestrated by Temporal        |                 | alpha  | >=0.0.3-dev.1    |
+| [Wasm](./runtimes/wasm)         | Execute WebAssembly functions                              |                 | alpha  | >=0.0.3-dev.6    |
```

Comment:
Not convince that the columns really add value, I would push to have something like https://learn.golem.cloud/docs/building-templates where only the tiers and some links are shown